example/lambdas.yml

```diff
@@ -136,7 +136,6 @@ SfSnsReport:
 SyncGranule:
   handler: index.handler
   timeout: 300
-  logToElasticSearch: true
```

Comment:
@ifestus Is this on purpose? What is the reason for this change?

Reply:
Good catch. I removed it so I could set `logToSharedDestination` to true, since we can't set more than one log destination for a log group. I'll add that line back :+1: